# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: 2590d04ff745
# Full Hash: 2590d04ff745a26f356936c6943b5863262e8770
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-03-12 15:32:35
# Description:
#   Bug 1695782 - Verify that YUV texture step is non-zero. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D107778
# ==============================================================================

diff -r dcc686463127 -r 2590d04ff745 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Thu Mar 11 20:47:37 2021 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Thu Mar 11 20:48:05 2021 +0000
@@ -202,3 +202,4 @@
 load 1679477-1.html
 skip-if(Android) pref(dom.disable_open_during_load,false) load 1685009-1.html
 load 1694783-1.html
+load 1695782-1.html